guide/xml/portgroups.xml

```diff
@@ -10,27 +10,33 @@
   group of portfiles, in order that those portfiles can be expressed as simply as possible with
   minimum redundancy.</para>
 
-<para>See the following folder for PortGroup definitions:</para>
+  <para>See the following folder for PortGroup definitions:</para>
   <para>
     <filename>${prefix}/var/macports/sources/rsync.macports.org/release/tarballs/ports/_resources/port1.0/group/</filename>
   </para>
 
   <para>A sample listing follows:</para>
     <para>
-    <programlisting><prompt>%%</prompt> <userinput>ls -1 /opt/local/var/macports/sources/rsync.macports.org/release/tarballs/ports/_resources/port1.0/group/</userinput></programlisting>
+      <programlisting><prompt>%%</prompt> <userinput>ls -1 /opt/local/var/macports/sources/rsync.macports.org/macports/release/tarballs/ports/_resources/port1.0/group/</userinput></programlisting>
```

Comment:
FYI, that wasn't a typo, but it was an old value and you're right to change it as you did.

Reply:
Thanks.